x11-packages/gw/build.sh

```diff
@@ -55,7 +55,7 @@ termux_step_make() {
 	unzip -o "${TERMUX_PKG_CACHEDIR}/skia-${TERMUX_ARCH}.zip"
 	cd ../../
 
-	make -j "$TERMUX_PKG_MAKE_PROCESSES"
+	LDFLAGS="${LDFLAGS} -L${TERMUX_PREFIX}/lib" make -j "$TERMUX_PKG_MAKE_PROCESSES"
```

Comment:
Is this actually needed?

build-package.sh:642 initialises the `LDFLAGS` environment variable, which is called before `termux_step_pre_configure`

Reply:
```suggestion
	make -j "$TERMUX_PKG_MAKE_PROCESSES"
```

Reply:
Once this change is not needed, please undo this